Resources/Private/CodeTemplates/Extbase/Partials/Form/SlugProperty.phpt

```diff
@@ -0,0 +1,13 @@
+[
+    'type' => 'slug',
+    'generatorOptions' => [
+        'fields' => ['title', 'nav_title'],
```

Comment:
why `nav_title` here? That is only used in the pages table

Reply:
I don't know what would be the best default value because the extension builder does not know which field(s) should be used.

Reply:
The best would be to add an additional field in the gui for it but let's leave it for now just with "title"